plugin-server/src/main/ingestion-queues/session-recording-v2/kafka/parser.ts

```diff
@@ -0,0 +1,99 @@
+import { promisify } from 'node:util'
+const { unzip } = require('node:zlib')
+import { Message } from 'node-rdkafka'
+
+import { PipelineEvent, RawEventMessage, RRWebEvent } from '../../../../types'
+import { status } from '../../../../utils/status'
+import { KafkaMetrics } from './metrics'
+import { ParsedMessageData } from './types'
+
+const GZIP_HEADER = Buffer.from([0x1f, 0x8b, 0x08, 0x00])
+const do_unzip = promisify(unzip)
```

Comment:
same with camelCasing it 
```suggestion
const decompressWithGzip = promisify(unzip)
```